pkg/interface/package.json

```diff
@@ -9,7 +9,7 @@
     "@reach/menu-button": "^0.10.5",
     "@reach/tabs": "^0.10.5",
     "@tlon/indigo-light": "^1.0.3",
-    "@tlon/indigo-react": "1.2.6",
+    "@tlon/indigo-react": "urbit/indigo-react#lf/1.2.8",
```

Comment:
The secret Liam changes ...

Reply:
PR'd https://github.com/urbit/indigo-react/pull/71